public/app/features/alerting/unified/api/buildInfo.test.ts

```diff
@@ -36,7 +36,11 @@ describe('buildInfo', () => {
         })
       );
 
-      const response = await fetchDataSourceBuildInfo({ url: '/datasource/proxy', name: 'Prometheus' });
+      const response = await discoverDataSourceFeatures({
+        url: '/datasource/proxy',
+        name: 'Prometheus',
+        type: 'prometheus',
```

Comment:
Do we have an `enum` for this? I'd love for the compiler to tell us when we're sending an unknown query string value here